deepmd/dpmodel/descriptor/se_atten_v2.py

```diff
@@ -158,7 +161,7 @@ def serialize(self) -> dict:
     def deserialize(cls, data: dict) -> "DescrptSeAttenV2":
         """Deserialize from dict."""
         data = data.copy()
-        check_version_compatibility(data.pop("@version"), 1, 1)
+        check_version_compatibility(data.pop("@version"), 2, 2)
```

Comment:
For dp/tf, I make it compatible in https://github.com/deepmodeling/deepmd-kit/pull/3958/commits/27434e7167e1f7128427466b806e29ec96d5f788. But for old trained `model.pt`, I found making it compatible with new code may make code unnecessarily confusing. (check state_dict keys and add extra logics.) Maybe we leave it as a breaking change?

Reply:
You just need to use `self.register_parameter("bias", None)` instead of `self.bias = None`, so `load_state_dict` can correctly override it. See https://github.com/pytorch/pytorch/blob/cb2bce98de5bba1c5415a08d7b17c608f6c77b62/torch/nn/modules/activation.py#L1084. By the way, other parameters like `idt` should follow the same way to have the maximum compatibility.

Reply:
Oh I found this issue: https://github.com/pytorch/pytorch/issues/8104

Reply:
It seems this is the only way:
```py
import torch

class TestParameter(torch.nn.Module):
    def __init__(self):
        super().__init__()
        self.bias = None

    def load_state_dict(self, state):
        if self.bias is None and "bias" in state:
            self.bias = torch.nn.Parameter(data=state["bias"])
        super().load_state_dict(state)

m=TestParameter()
m.load_state_dict({"bias": torch.zeros(3)})
print(m.bias)
```

Reply:
It may be better to make a breaking change? One can convert it to the DP model using the old version, and then back to the PyTorch model using the new version.

Reply:
> It may be better to make a breaking change? One can convert it to the DP model using the old version, and then back to the PyTorch model using the new version.

I agree.